nx-X11/config/cf/xf86.tmpl

```diff
@@ -1,128 +0,0 @@
-XCOMM $Xorg: xf86.tmpl,v 1.3 2000/08/17 19:41:48 cpqbld Exp $
```

Comment:
`nx-X11/config/cf/Server.tmpl` still references `xf86.tmpl` (and has a `XFree86Version` define.)

Other places which use `XFree86Version`:
  - `nx-X11/config/cf/Imakefile`
  - `nx-X11/programs/Xserver/GL/mesa/X/Imakefile`
  - `nx-X11/programs/Xserver/Imakefile`
  - `nx-X11/programs/Xserver/include/Imakefile`
  - `nx-X11/programs/Xserver/mi/Imakefile`
  - `nx-X11/programs/Xserver/Xext/Imakefile`

Places with `XFree86VersionString`:
  - `nx-X11/config/Imakefile`